### PR TITLE
#1124-wp-detail-revamp-changes-tab

### PR DESCRIPTION
**Pull request by LOTaher**
## Changes

Added the (soon to come) changes tab's content

## Notes

Removed everything else from the wp detail page. The content will just be put into the Changes tab

I did edit `ChangesList.tsx`, I do not think that should cause any problems though.

## Screenshots

image

image

## To Do

- [ ] Implement the tab content onto the actual tab once it's created

## Checklist

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [ ] No merge conflicts
- [ ] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes # (issue #1124)

### CODE REVIEW

**Comment by LOTaher** [comment]
Not sure how to properly fix lines 45-47 linting errors. If I delete those lines altogether, how should I go about changing the methods.

**Comment by LOTaher** [comment]
image
Because I am using the ChangesList component with BulletList, there is still a title above the bullet points.

File: src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/WorkPackageViewContainer.tsx [hunk 1]
```diff
@@ -163,53 +155,7 @@ const WorkPackageViewContainer: React.FC<WorkPackageViewContainerProps> = ({
         ]}
         actionButton={projectActionsDropdown}
       />
-      <WorkPackageDetails workPackage={workPackage} />
```
Comment (Peyton-McKee) [inline src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/WorkPackageViewContainer.tsx:166]: you should undo all the changes you made to this file I think...

**Review by Peyton-McKee — CHANGES_REQUESTED** [review]
nice work! You dont need to delete the other stuff, once reids PR is merged all that stuff will still be used. Im also not sure what were gonna do about the header. I think well wait and see what it looks like with the tab and then adjust from there

File: src/frontend/src/components/BulletList.tsx [hunk 2]
```diff
@@ -30,9 +32,12 @@ const BulletList: React.FC<BulletListProps> = ({ title, headerRight, list, order
     builtList = <ol style={styles.bulletList}>{listPrepared}</ol>;
   }
   return (
-    <PageBlock title={title} headerRight={headerRight} defaultClosed={defaultClosed}>
+    <>
+      <Typography variant="h3" sx={{ fontFamily: 'Oswald,sans-serif' }}>
```
Comment (RChandler234) [inline src/frontend/src/components/BulletList.tsx:36]: I don't think you need this fontFamilty stuff, typography on its own should set the font correctly

File: src/frontend/src/components/BulletList.tsx [hunk 3]
```diff
@@ -5,6 +5,8 @@
 
 import { ReactNode } from 'react';
 import PageBlock from '../layouts/PageBlock';
+import { fontFamily } from '@mui/system';
```
Comment (RChandler234) [inline src/frontend/src/components/BulletList.tsx:8]: remove unused imports

File: src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/WorkPackageViewContainer.tsx [hunk 4]
```diff
@@ -8,13 +8,8 @@ import { Link } from 'react-router-dom';
 import { isGuest, WbsElementStatus, WorkPackage } from 'shared';
```
Comment (RChandler234) [inline src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/WorkPackageViewContainer.tsx:8]: remove unused import

**Review by RChandler234 — CHANGES_REQUESTED** [review]

**Comment by LOTaher** [comment]
A new PR will be made under a branch that's base has the tabs implemented

**Pull request closed without merging**